cmd/podman/machine/create.go

```diff
@@ -0,0 +1,107 @@
+package machine
+
+import (
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/machine"
+	"github.com/containers/podman/v3/pkg/machine/qemu"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:               "create [options] NAME",
+		Short:             "Create a vm",
+		Long:              "Create a virtual machine for Podman to run on. Virtual machines are used to run Podman on Macs. ",
```

Comment:
podman-remove machine create will work on more then just Macs?  I would just remove `on Macs`

Reply:
Concur that we should remove any mention of Mac in the doc text. We can mention that the primary target is OS X in the manpages if requires, but this should also work on Linux.